launcher/launcher_MAC.command

```diff
@@ -1,4 +1,16 @@
 cd "$( dirname "$0" )"
+
+if ! type -p java > /dev/null 2>&1; then
+  echo "Java is not installed."
+  exit
```

Comment:
Please don't use `exit` in these cases. The user needs to see the error when they've double-clicked on the command. Use the same "Press Enter to exit..." thing we do at the bottom, please.